pkg/docs/sql_schema.md

```diff
@@ -95,4 +97,78 @@ Example query to look at all the series:
 -----------+---------+------------+-------
          4 | {3,4}   | dev        | pinky
          5 | {5,6}   | production | brain
-```
\ No newline at end of file
+```
+
+## Series Selectors
+
+We have added simple-to-use series selectors for filtering series in either of the two views above.
+
+### Containment
+
+To test whether a series contains a json fragment you can use the `@>` containment operator.
+
+For example to find all metrics on in the dev namespace and on node pinky, run:
+
+```SQL
+SELECT *
+FROM cpu_usage  u
+WHERE labels @> jsonb '{"namespace":"dev", "node": "pinky"'}
+```
+
```

Comment:
perhaps a sample output?

> Sample output
> ``` 
>  series_id | labels  | namespace  | node  | region
>-----------+---------+------------+-------+--------
>          4 | {3,4,5} | dev        | pinky | East
>          5 | {3,4,5} | dev        | pinky | West
> ```
To demonstrate that unspecified matchers can be different